### PR TITLE
Selectable items

### PR DESCRIPTION
**Pull request by hjwebb01**
Base functionality for a selectable drop with 1 example included

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -1,8 +1,6 @@
 body {
   background-color: rgb(75, 0, 60);
 }
-h1
```
Comment (ejshakman) [inline static/css/styles.css:4]: could you put these 2 lines in your code? I may have accidentally removed them when I was fixing the merge conflict
h1
{

File: static/css/styles.css [hunk 2]
```diff
@@ -24,7 +22,111 @@ p {
   color:blanchedalmond;
   font-family:palatino;
 }
+*{
```
Comment (ejshakman) [inline static/css/styles.css:25]: could you please put in some comments for your additions from this line and down?

File: templates/index.html [hunk 3]
```diff
@@ -5,10 +5,54 @@
...
         <script src="{{ url_for('static', filename='js/main.js') }}"></script>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
     </head>
+    
     <body>
-        <h1>Mix Master: Your Personal Bartender</h1>
+        <h1>Mix Master</h1>
         <h2>Mojito</h2>
         <p>The quick brown fox jumps over the lazy dog. THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG.</p>
+    <div class="container">
```
Comment (ejshakman) [inline templates/index.html:15]: could you please put in some comments from this line and down?

File: static/css/styles.css (hunk 1 above)
Comment (ejshakman) [inline static/css/styles.css:4]: could you put these 2 lines in your code? I may have accidentally removed them when I was fixing the merge conflict
h1
{

**Review by ejshakman — CHANGES_REQUESTED** [review]
Just requesting a small addition to styles.css and some comments on your code in the other files

**Review by Archaversine — APPROVED** [review]

**Review by ejshakman — APPROVED** [review]
Looks good, thanks!

**Pull request merged**